libraries/DataFlash/LogStructure.h

```diff
@@ -1106,10 +1107,10 @@ struct PACKED log_DSTL {
 #define QUAT_UNITS  "s????"
 #define QUAT_MULTS  "F????"
 
-#define CURR_LABELS "TimeUS,Volt,VoltR,Curr,CurrTot,Temp,Res"
-#define CURR_FMT    "Qffffcf"
-#define CURR_UNITS  "sv?A?Ow"
-#define CURR_MULTS  "F??????"
+#define CURR_LABELS "TimeUS,Volt,VoltR,Curr,CurrTot,EnrgTot,Temp,Res"
+#define CURR_FMT    "Qfffffcf"
+#define CURR_UNITS  "sv?A??Ow"
```

Comment:
Please add a proper unit. Adding a ? is NOT a solution.

Reply:
see https://github.com/ArduPilot/ardupilot/blob/master/libraries/DataFlash/LogStructure.h line 74